config/cms.php

```diff
@@ -329,4 +329,21 @@
 
     'forceBytecodeInvalidation' => true,
 
+    /*
+    |--------------------------------------------------------------------------
+    | Backend login remember
+    |--------------------------------------------------------------------------
+    |
+    | Define live duration of backend sessions :
+    |
+    | true  - session never expire (cookie expiration in 5 years)
+    |
+    | false - session have a limited time (see session.lifetime)
+    |
+    | null  - The form login display a checkbox that allow user to choose
+    |         wanted behavior
+    |
+    */
+
+    'backendLoginRemember' => true,
```

Comment:
Please rename to `backendForceRemember`, and move this to just below https://github.com/octobercms/october/blob/master/config/cms.php#L52